libraries/core-react/src/Dialog/index.ts

```diff
@@ -3,16 +3,16 @@ import { Actions } from './Actions'
 import { Title } from './Title'
 import { CustomContent } from './CustomContent'
 
-type DialogTypes = typeof BaseComponent & {
+type DialogProps = typeof BaseComponent & {
   Actions: typeof Actions
   Title: typeof Title
   CustomContent: typeof CustomContent
 }
 
-const Dialog = BaseComponent as DialogTypes
+const Dialog = BaseComponent as DialogProps
 
 Dialog.Actions = Actions
 Dialog.Title = Title
 Dialog.CustomContent = CustomContent
 
-export { Dialog }
+export { Dialog, DialogProps }
```

Comment:
Didn't we agree to export the actual `DialogProps` here? @wenche @vnys

Reply:
Yes we did